test/utilities/testsUtils.ts

```diff
@@ -0,0 +1,222 @@
+  testsSection: ViewSection,
+  actionLabel: string
+): Promise<TreeItem[]> {
+  let testsItems = (await testsSection.getVisibleItems()) as TreeItem[];
+  await browser.keys(['Escape']);
+
+  // If the tests did not show up, click the refresh button on the top right corner of the Test sidebar
+  for (let x = 0; x < 3; x++) {
+    if (testsItems.length === 1) {
+      await testsSection.elem.click();
+      const refreshAction = await testsSection.getAction(actionLabel);
+      if (!refreshAction) {
+        fail('Could not find refresh tests action button');
+      }
+      await refreshAction.elem.click();
+      await pause(Duration.seconds(10));
+      testsItems = (await testsSection.getVisibleItems()) as TreeItem[];
+    } else if (testsItems.length === expectedNumTests) {
+      break;
+    }
+  }
+
+  return testsItems;
+}
+
+export async function getTestsSection(workbench: Workbench, type: string) {
+  const sidebar = workbench.getSideBar();
+  const sidebarView = sidebar.getContent();
+  const testsSection = await sidebarView.getSection(type);
+  await expect(testsSection.elem).toBeDefined();
+  return testsSection;
+}
+
+/**
+ * Runs a test case from the sidebar and returns the test result.
+ * *
+ * @param {Workbench} workbench - The workbench instance used to interact with the sidebar and views.
+ * @param {string} testSuite - The name of the test suite from which to run the test (e.g., 'APEX TESTS', 'LWC TESTS').
+ * @param {string} testName - The name of the specific test case to run.
+ * @param {string} actionLabel - The label of the action button to click (e.g., 'SFDX: Run Apex Tests', 'Debug').
+ *
+ * @example
+ * const result = await runTestCaseFromSideBar(
+ *   myWorkbench,
+ *   'APEX TESTS',
+ *   'MyApexTestCase',
+ *   'SFDX: Run Apex Tests'
+ * );
+ * console.log(result); // Outputs the result from the Apex test run
+ */
+export async function runTestCaseFromSideBar(
+  workbench: Workbench,
+  testSuite: string,
+  testName: string,
+  actionLabel: string
+): Promise<string | undefined> {
+  log(`Running ${testSuite} - ${testName} - ${actionLabel} from SideBar`);
+  const testingView = await workbench.getActivityBar().getViewControl('Testing');
+  await expect(testingView).not.toBeUndefined();
+
+  // Open the Test Sidebar
+  const testingSideBarView = await testingView?.openView();
+  await expect(testingSideBarView).toBeInstanceOf(SideBarView);
+
+  // Select test
+  const testSection = await getTestsSection(workbench, testSuite);
+  const testItem = (await testSection.findItem(testName)) as TreeItem;
+  await expect(testItem).toBeDefined();
+  await testItem.select();
+
+  // Click button to run test
+  const actionButton = await testItem.getActionButton(actionLabel);
+  await expect(actionButton).toBeDefined();
+  await actionButton?.elem.click();
+
+  if (actionLabel.includes('Debug')) {
+    await pause(Duration.seconds(10));
+
+    // Continue with the debug session
+    await continueDebugging();
+  }
```

Comment:
Is this block needed?  I see that the `runTestCaseFromSideBar()` helper function currently isn't being used for the DebugApexTests and DebugLWCTests E2E tests.

Reply:
Oh good catch, I was trying some stuff and forgot to remove this